### PR TITLE
Adding an Alteration Amount column to the download data table

### PR DESCRIPTION
**Pull request by noodleboi929**
Fix cBioPortal/cbioportal#8508

Adding an alteration count column to the download data file from the cancer types summary graph. The alteration count column shows how many cases were actually mutated based on the graph and given percentage.
image

### CODE REVIEW

**Comment by noodleboi929** [comment]
Adding an alteration count column to the download data file from the cancer types summary graph.
image

File: src/pages/resultsView/cancerSummary/CancerSummaryChart.tsx [hunk 1]
```diff
@@ -707,7 +710,12 @@ export class CancerSummaryChart extends React.Component<
         let flatdata = this.convertDataToDownloadData(this.props.data);
         return tsvFormatRows(
             [
-                [this.props.xAxisString, this.yAxisLabel, 'Alteration Type'],
+                [
+                    this.props.xAxisString,
+                    this.yAxisLabel,
+                    'Alteration Type',
+                    'Alteration Amount',
```
Comment (dippindots) [inline src/pages/resultsView/cancerSummary/CancerSummaryChart.tsx:717]: Let's call this `Alteration Count`

File: src/pages/resultsView/cancerSummary/CancerSummaryChart.tsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ import {
     DownloadControls,
     DataType,
     pluralize,
-} from 'cbioportal-frontend-commons';
+} from '../../../../packages/cbioportal-frontend-commons';
```
Comment (dippindots) [inline src/pages/resultsView/cancerSummary/CancerSummaryChart.tsx:32]: we don't need to change this, right?

File: src/pages/resultsView/cancerSummary/CancerSummaryContainer.tsx [hunk 3]
```diff
@@ -6,14 +6,14 @@ import { MSKTabs, MSKTab } from 'shared/components/MSKTabs/MSKTabs';
...
-import { Gene } from 'cbioportal-ts-api-client';
+import { Gene } from '../../../../packages/cbioportal-ts-api-client';
 import './styles.scss';
 import {
     getAlterationCountsForCancerTypesByGene,
     getAlterationCountsForCancerTypesForAllGenes,
 } from '../../../shared/lib/alterationCountHelpers';
 import OqlStatusBanner from '../../../shared/components/banners/OqlStatusBanner';
-import { getRemoteDataGroupStatus } from 'cbioportal-utils';
+import { getRemoteDataGroupStatus } from '../../../../packages/cbioportal-utils';
```
Comment (dippindots) [inline src/pages/resultsView/cancerSummary/CancerSummaryContainer.tsx:16]: we don't need to change these too

**Review by dippindots — COMMENTED** [review]
Thanks @noodleboi929, this pr looks good, I had some minor suggestions, please see my comments.

**Review by dippindots — APPROVED** [review]
Looks good to me!

**Pull request merged**